TodoList/Tests/TodoList.Api.Tests/Controllers/ListItemsControllerTest.cs

```diff
@@ -1,178 +1,187 @@
+using TodoList.Api.Services;
+using TodoList.Api.Tests.Helpers;
+using TodoList.Contracts.Models;
+using TodoList.Contracts.Repositories;
+
+namespace TodoList.Api.Tests.Controllers
+{
+    [TestFixture]
+    public class ListItemsControllerTest
+    {
+        private IListItemRepository _repository;
+        private readonly Guid _guidOfFirstItem = new Guid("98DBDE18-639E-49A6-8E51-603CEB2AE92D");
+        private readonly Guid _guidOfSecondItem = new Guid("1C353E0A-5481-4C31-BD2E-47E1BAF84DBE");
+        private readonly Guid _guidOfThirdItem = new Guid("D69E065C-99B1-4A73-B00C-AD05F071861F");
+        private ListItemsController _controller;
+        private IListItemUrlGenerator _urlGenerator;
+
+        [SetUp]
+        public void Init()
+        {
+            _repository = Substitute.For<IListItemRepository>();
+            _urlGenerator = Substitute.For<IListItemUrlGenerator>();
+
+            _controller = new ListItemsController(_repository, _urlGenerator)
+            {
+                Configuration = new HttpConfiguration(),
+                Request = new HttpRequestMessage(),
+            };
+        }
+
+        [Test]
+        public void DeleteAsync_ReturnsCorrectResponse()
+        {
+            var expectedListItem = new ListItem { Id = _guidOfSecondItem, Text = "giraffe" };
+            _repository.DeleteAsync(Guid.Empty).Returns(expectedListItem);
+            
+            var actionResult = _controller.DeleteAsync(Guid.Empty).Result;
+            var responseMessage = actionResult.ExecuteAsync(CancellationToken.None).Result;
+            ListItem actualListItem;
+            responseMessage.TryGetContentValue(out actualListItem);
+
+            Assert.That(responseMessage.StatusCode, Is.EqualTo(HttpStatusCode.OK));
+            Assert.That(actualListItem, Is.EqualTo(expectedListItem).UsingListItemComparer());
+        }
+
+        [Test]
+        public void GetAsync_ById_ReturnsCorrectResponse()
+        {
+            var expectedListItem = new ListItem { Id = _guidOfFirstItem, Text = "text" };
+            _repository.GetAsync(_guidOfFirstItem).Returns(expectedListItem);
+
+            var actionResult = _controller.GetAsync(_guidOfFirstItem).Result;
+            var responseMessage = actionResult.ExecuteAsync(CancellationToken.None).Result;
+            ListItem actualListItem;
+            responseMessage.TryGetContentValue(out actualListItem);
+
+            Assert.That(responseMessage.StatusCode, Is.EqualTo(HttpStatusCode.OK));
+            Assert.That(actualListItem, Is.EqualTo(expectedListItem).UsingListItemComparer());
+        }
+
+        [Test]
+        public void GetAsync_ById_ReturnsCorrectErrorResponse()
+        {
+            var actionResult = _controller.GetAsync(Guid.Empty).Result;
+            var responseMessage = actionResult.ExecuteAsync(CancellationToken.None).Result;
+
+            Assert.That(responseMessage.StatusCode, Is.EqualTo(HttpStatusCode.BadRequest));

```

Comment:
The error message (or at least the message's key) should be asserted here as well.